src/components/Sections/WhyDevlake.tsx

```diff
@@ -102,10 +102,10 @@ function Supports() {
         className="text-label18 font-inter text-neutral-500 mt-6
         sm:text-label16 sm:mt-4"
       >
-        <C.InlineLink link="https://devlake.apache.org/docs/Overview/Roadmap">
-          More data sources
-        </C.InlineLink>{" "}
-        coming soon...
+        Check out all {" "}
+        <C.InlineLink link="https://devlake.apache.org/docs/next/Overview/SupportedDataSources">
```

Comment:
It should point to the stable doc `https://devlake.apache.org/docs/Overview/SupportedDataSources`, not the next doc